File: app/src/main/java/mariavv/fitnesspal/data/db/jounal/Journal.java
package mariavv.fitnesspal.data.db.jounal;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import java.util.Date;

@Entity
public class Journal {
    @PrimaryKey
    public int id;
    public int meal_num;
    public Date date;
    public int handbook_id;
    public int mass;
}

File: app/src/main/java/mariavv/fitnesspal/data/db/jounal/Journal.kt
package mariavv.fitnesspal.data.db.jounal

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import java.util.*

@Entity
class Journal {
    @PrimaryKey
    var id: Int = 0
    var meal_num: Int = 0
    var date: Date? = null
    var handbook_id: Int = 0
    var mass: Int = 0
}


